pasteurize.nix

```diff
@@ -124,6 +124,12 @@
                     inherit (asserted.bee) system pkgs;
                     inherit (asserted.bee.pkgs) config; # nixos modules don't load this
                   };
+                  # seemlessly integrate hm if desired
+                  imports =
+                    []
+                    ++ l.optionals (asserted.bee ? home) [
+                      config.bee.home.nixosModules.home-manager
+                    ];
```

Comment:
I'm pretty certain that this is not enough and something like "use global packages" is required.

@gtrunsec can you inform the details about the correct configuration?

Reply:
https://github.com/GTrunSec/hive/blob/main/nix/guangtao/nixosConfigurations/desktop/default.nix#L46